std/conv.d

```diff
@@ -5504,86 +5506,160 @@ Params:
 
 Returns:
     a $(D string), a $(D wstring) or a $(D dstring), according to the type of hexData.
- */
-template hexString(string hexData)
+*/
+template hexString(alias hexData)
 if (hexData.isHexLiteral)
 {
-    immutable hexString = hexStrImpl(hexData);
+    import std.array: array;
+    enum hexString = HexStringRange!(typeof(hexData))(hexData).array;
 }
-
-/// ditto
-template hexString(wstring hexData)
-if (hexData.isHexLiteral)
+///
+@safe pure unittest
 {
-    immutable hexString = hexStrImpl(hexData);
+    static assert(hexString!"304A314B" == "0J1K");
+    static assert(hexString!"304A314B"w == "0J1K"w);
+    static assert(hexString!"304A314B"d == "0J1K"d);
 }
 
-/// ditto
-template hexString(dstring hexData)
-if (hexData.isHexLiteral)
+/**
+Does exactly the same thing as the `hexString` template but at run-time.
```

Comment:
Why not, I planed to do it but  didn't because after lokking at the result with Vlad's  ddoc builder I've realized that since it's for an overload the user has just to scroll a bit up.